2020-04-introducing-mdjs-live-demos-everywhere/README.md

```diff
@@ -0,0 +1,360 @@
+for (const story of stories) {
+  const storyEl = rootNode.querySelector(`[mdjs-story-name="${story.key}"]`);
+  storyEl.story = story.story;
+  storyEl.code = story.code;
+}
+```
+
+All of this happens under the hood for you 🤗
+
+## Where can you use mdjs?
+
+### You can use it locally via es-dev-server
+
+This will create a github like markdown view for all your local markdown files including live demos.
+
+![es-dev-server screenshot](https://raw.githubusercontent.com/open-wc/blog-posts/feat/mdjs/2020-04-introducing-mdjs-live-demos-everywhere/images/es-dev-server-screenshot.png)
+
+- Install dependency `npm i -D es-dev-server`
+
+- Add to your `package.json`:
+
+  ```json
+  "scripts": {
+    "start": "es-dev-server",
+  }
+  ```
+
+- Create a `es-dev-server.config.js` in the root of your repo.
+
+  ```js
+  const { mdjsTransformer } = require('@mdjs/core');
+
+  module.exports = {
+    nodeResolve: true,
+    open: 'README.md',
+    watch: true,
+    responseTransformers: [mdjsTransformer],
+  };
+  ```
+
+After executing `npm run start` you can happily browse you live documentation via [http://localhost:8000/README.md](http://localhost:8000/README.md).
+
+You can see an example setup in the [demo-wc-card repo](https://github.com/daKmoR/demo-wc-card).
+
+### You can use it via storybook
+
+If you want to work on individual components or get a list of all demos you can use storybook.
+
+![storybook screenshot](https://raw.githubusercontent.com/open-wc/blog-posts/feat/mdjs/2020-04-introducing-mdjs-live-demos-everywhere/images/storybook-screenshot.png)
+
+- Install dependency `npm i -D @open-wc/demoing-storybook`
+
+- Add to your `package.json`:
+
+  ```json
+  "scripts": {
+    "storybook": "start-storybook --experimental-md-docs",
+  }
+  ```
+
+- Adjust your `.storybook/main.js` to load markdown files
+
+  ```js
+  module.exports = {
+    stories: ['../README.md', '../docs/**/*.md'],
+    esDevServer: {
+      nodeResolve: true,
+      watch: true,
+      open: true,
+    },
+  };
+  ```
+
+- Add to every markdown file that should be in storybook a name via
+
+  ```js
+  export default {
+    title: 'My Group/My Awesome Component',
+  };
+  ```
+
+With that, you are good to go.
+No additional changes to any files are needed a plugin will take care of it by converting your markdown files to the support mdx format of storybook.
+
+For more detailed information please see [https://open-wc.org/demoing-storybook/](https://open-wc.org/demoing-storybook/).
+
+### Show it on github
+
+Github supports markdown out of the box and with this format we can go one step further.
+
+![github screenshot](https://raw.githubusercontent.com/open-wc/blog-posts/feat/mdjs/2020-04-introducing-mdjs-live-demos-everywhere/images/github-screenshot.png)
+
+As it's not (yet - let's ask 😬) supported by github directly you will need a chrome extension called [mdjs-viewer](https://chrome.google.com/webstore/detail/mdjs-viewer/ifkkmomkjknligelmlcnakclabgohafe).
```

Comment:
I feel like the 'let's ask' comes off a little too eager here imo. If you want this to be supported in github, I'd suggest making an issue or reaching out to someone on twitter with a link to your blogpost. I'd leave it out of the blog itself. Or word it differently.

Reply:
makes sense so just remove it?

```suggestion
As it's not supported by github directly you will need a chrome extension called [mdjs-viewer](https://chrome.google.com/webstore/detail/mdjs-viewer/ifkkmomkjknligelmlcnakclabgohafe).
```